src/features/accounts/data/funded-account.ts

```diff
@@ -0,0 +1,34 @@
+import { algorandClient } from '@/features/common/data/algo-client'
+import {
+  BETANET_FEE_SINK_ADDRESS,
+  FNET_FEE_SINK_ADDRESS,
+  fnetId,
+  MAINNET_FEE_SINK_ADDRESS,
+  selectedNetworkAtomId,
+  TESTNET_FEE_SINK_ADDRESS,
+} from '@/features/network/data'
+import { settingsStore } from '@/features/settings/data'
+import { atom } from 'jotai'
+import { Address } from './types'
+import { betanetId, localnetId, mainnetId, testnetId } from '@/features/network/data/types'
+
+export const fundedAccountAtom = atom<Promise<Address | undefined>>(async () => {
```

Comment:
I extracted this outside of network, as it felt like it better fit into account.
This atom imperatively calls the settingsStore to get the selected network, however the data held within is stored in the main data jotai store.